common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -5899,4 +5900,12 @@ object CelebornConf extends Logging {
       .doubleConf
       .checkValue(v => v > 0.0 && v <= 1.0, "Should be in (0.0, 1.0].")
       .createWithDefault(1)
+
+  val CLIENT_QUOTA_INTERRUPT_SHUFFLE_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.quota.interruptShuffle.enabled")
```

Comment:
Perhaps we should rename the configuration to celeborn.quota.interruptShuffle.enabled, as 'quota' represents an independent configuration namespace. It would also be appropriate to move this setting to the quota section.